neopia/neosoco.py

```diff
@@ -739,5 +745,20 @@ def buzzer_by_port(self, port='in1'):
             # Map to 0~65 from 0~100(max), it's same as Entry
             value = self._convert_scale_from_input_port(port, 65)
             self.write(Neosoco.NOTE, value)
+        else:
+            raise TypeError
+    
+    def buzzer_stop(self):
+        self.write(Neosoco.NOTE, 0)
+
+    def remote_button(self, button='1'):

```

Comment:
In this commit this API's additions is not included to your commit because it's not a new work.